src/main/java/com/sparta/popupstore/domain/review/dto/request/ReviewSaveRequestDto.java

```diff
@@ -1,4 +1,23 @@
 package com.sparta.popupstore.domain.review.dto.request;
 
+import com.sparta.popupstore.domain.popupstore.entity.PopupStore;
+import com.sparta.popupstore.domain.review.entity.Review;
+import com.sparta.popupstore.domain.user.entity.User;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
 public class ReviewSaveRequestDto {
+  private int star;
+  private String content;
+
+  public Review toReviewSaveRequestDto(User user, Long id) {
+    return Review.builder()
+        .user(user)
+        .popupStore(PopupStore.builder().id(id).build())
+        .star(star)
+        .contents(content)
+        .build();
+  }
```

Comment:
```suggestion
  public Review toEntity(User user, PopupStore popupStore) {
    return Review.builder()
        .user(user)
        .popupStore(popupStore)
        .star(this.star)
        .contents(this.content)
        .build();
  }
```
메서드 이름이 잘못되었습니다.
popupStore도 user처럼 파라미터로 입력 받는 게 맞습니다.
클래스의 필드를 쓸 때는 this.을 붙이는게 좋습니다.
content와 contents 이름을 통일해주세요.